Microsoft.Toolkit.Uwp.SampleApp/SamplePages/LottieAnimationView/LottieAnimationViewPage.xaml.cs

```diff
@@ -0,0 +1,131 @@
+        {
+            InitializeComponent();
+        }
+
+        public void OnXamlRendered(FrameworkElement control)
+        {
+            Frame rootFrame = Window.Current.Content as Frame;
+            _lottieAnimationView = control.FindChildByName("Container") as LottieAnimationView;
+            _lottieAnimationView.Loaded += (s, e) =>
+            {
+                _rangeSelector = VisualTree.FindDescendant<RangeSelector>(Window.Current.Content as Frame);
+                UpdateRangeSelector();
+            };
+
+            _lottieAnimationViewGrid = control.FindChildByName("LottieAnimationViewGrid") as Grid;
+        }
+
+        protected override void OnNavigatedTo(NavigationEventArgs e)
+        {
+            base.OnNavigatedTo(e);
+
+            Shell.Current.RegisterNewCommand("Resume/Pause", ResumePauseButton_Click);
+            Shell.Current.RegisterNewCommand("Open File", OpenFileButton_Click);
+            Shell.Current.RegisterNewCommand("Background White", BackgroundColorButton_Click);
+        }
+
+        private void UpdateRangeSelector()
+        {
+            if (_rangeSelector != null)
+            {
+                _rangeSelector.Minimum = _lottieAnimationView.StartFrame;
+                _rangeSelector.Maximum = _lottieAnimationView.EndFrame;
+
+                var sample = DataContext as Sample;
+                var propDict = sample.PropertyDescriptor.Expando as IDictionary<string, object>;
+                (propDict["MinFrame"] as ValueHolder).Value = _rangeSelector.Minimum;
+                (propDict["MaxFrame"] as ValueHolder).Value = _rangeSelector.Maximum;
+            }
+        }
+
+        private void ResumePauseButton_Click(object sender, RoutedEventArgs e)
+        {
+            if (_lottieAnimationView.IsAnimating)
+            {
+                _lottieAnimationView.PauseAnimation();
+            }
+            else
+            {
+                if (_lottieAnimationView.TimesRepeated >= _lottieAnimationView.RepeatCount)
+                {
+                    _lottieAnimationView.PlayAnimation();
+                }
+                else
+                {
+                    _lottieAnimationView.ResumeAnimation();
+                }
+            }
+        }
+
+        private async void OpenFileButton_Click(object sender, RoutedEventArgs e)
+        {
+            var openPicker = new FileOpenPicker();
+            openPicker.FileTypeFilter.Add(".json");
+            var file = await openPicker.PickSingleFileAsync();
+
+            if (file != null)
+            {
+                await _lottieAnimationView.SetAnimationAsync(new StreamReader(await file.OpenStreamForReadAsync()));
+                UpdateRangeSelector();
+                _lottieAnimationView.PlayAnimation();
+            }
+        }
+
+        private void BackgroundColorButton_Click(object sender, RoutedEventArgs e)
+        {
+            var backgroundColorButton = (Button)e.OriginalSource;
```

Comment:
I followed this back around.  I think for now, we should just create two buttons, a 'Background Black' and 'Background White'.  We should provide a better system within the sample app to control the commands (either add a toggle type, or expose ways to hide/show commands, or specify parameters, etc...)

I created Issue #1960 to track this.

Reply:
Done.